sdk/communication/azure-communication-callingserver/azure/communication/callingserver/_call_connection.py

```diff
@@ -49,49 +81,69 @@ def hang_up(
 
     @distributed_trace()
     def cancel_all_media_operations(
-            self,
-            operation_context=None,  # type: Optional[str]
-            **kwargs  # type: Any
-        ): # type: (...) -> CancelAllMediaOperationsResult
+        self,
+        operation_context=None,  # type: Optional[str]
+        **kwargs  # type: Any
+    ): # type: (...) -> CancelAllMediaOperationsResult
 
         cancel_all_media_operations_request = CancelAllMediaOperationsConverter.convert(operation_context)
 
-        return self._call_connection_client.cancel_all_media_operations(
+        cancel_all_media_operations_result = self._call_connection_client.cancel_all_media_operations(
             call_connection_id=self.call_connection_id,
             cancel_all_media_operation_request=cancel_all_media_operations_request,
             **kwargs
         )
 
+        return CancelAllMediaOperationsResult._from_generated(cancel_all_media_operations_result)
+
     @distributed_trace()
     def play_audio(
-            self,
-            audio_file_uri,  # type: str
-            play_audio_options,  # type: PlayAudioOptions
-            **kwargs  # type: Any
-        ): # type: (...) -> PlayAudioResult
+        self,
+        audio_file_uri,  # type: str
+        play_audio_options,  # type: PlayAudioOptions
+        **kwargs  # type: Any
+    ): # type: (...) -> PlayAudioResult
 
         if not audio_file_uri:
             raise ValueError("audio_file_uri can not be None")
 
         if not play_audio_options:
             raise ValueError("options can not be None")
 
+        try:
```

Comment:
@satyakonmsft do we need to run some str type check here for **user passing uri** ?
For Java and C#, we didn't do it.

Reply:
yes we should check for valid url. we should add it for other sdks too